Ruby Basic/task4.rb

```diff
@@ -0,0 +1,21 @@
+puts 'input boolean true or false:'
+
+input_value = gets.chomp.downcase
+
+boolean_keywords = %w[true false]
+
+abort('You need enter true/false keyword.') unless boolean_keywords.include? input_value
+
+boolean_value = input_value.include?("true") ? true : false
```

Comment:
This operation `input_value.include?("true")` back to you boolean value, in this case you don't need ternary operator. And you verified `input_value` on previous step, that's why you need more strict comparison `==` instead `include?`.
Examples:
```ruby
  input_value.include?("true") ? true : false # => true
  input_value.include?("true") # => true
  input_value == 'true' # => true
```
In last example `'true' ` - string, not boolean, because script takes arguments as strings